packages/ionic/src/commands/cordova/run.ts

```diff
@@ -95,16 +124,6 @@ export class RunCommand extends CordovaCommand implements CommandPreRun {
       },
     ];
 
-    try {
```

Comment:
By doing this, I think you are throwing away the dynamic help output. `--help` uses `getMetadata()`

Reply:
You're right, that does happen. The problem I was trying to fix here was to get the right metadata based on whether `livereload` is enabled. It passes through to `ng serve` when `livereload` is enabled, and `ng build` when it's not. This is a problem if the build and serve options are different since the CLI filters unknown options. Any ideas?

Reply:
Yeah, there's not really a good solution here. Perhaps leave it the way you have it for now. I'm not too worried